files/semantic-ui.daterangepicker/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "semantic-ui.daterangepicker",
+  "repo": "BreadMaker/semantic-ui-daterangepicker",
+  "files": {
+    "include": ["daterangepicker.css", "daterangepicker.js", "daterangepicker.min.css", "daterangepicker.min.js"]
```

Comment:
Most of these files are not here https://github.com/BreadMaker/semantic-ui-daterangepicker/tree/v1.3.21